packages/keyring-api/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.4.0]
+
+### Added
+
+- Add `discoverAccounts` keyring method ([#266](https://github.com/MetaMask/accounts/pull/266))
+- Add `KeyringEventPayload` utility type to extract payload `KeyringEvent` enum ([#203](https://github.com/MetaMask/accounts/pull/203))
```

Comment:
Not sure if this should be listed since it's not exported

Reply:
This one is: https://github.com/MetaMask/accounts/blob/db4d703a61502b59c3e2cefd44158cca781a8192/packages/keyring-api/src/events.ts#L277

But indeed, `KeyringEvents` (plural), is not!